scripts/states/enemy/Walk.gd

```diff
@@ -0,0 +1,17 @@
+extends 'State.gd'
+
+
+func physics_update(delta: float) -> void:
+
+  if enemy.direction == -1.0:
```

Comment:
Not that it matters much here.
But it is considered good practice to bake the flipped around sprites into the spriteheet as it prevents any offset issues

Reply:
minor, will change later